code/modules/projectiles/guns/rifles.dm

```diff
@@ -2374,8 +2374,8 @@
 //L42A3 Battle Rifle
 
 /obj/item/weapon/gun/rifle/l42a3
```

Comment:
is there any reason why you didnt change the path name, just the name of the weapon? code is fine otherwise